kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/row/GCSToBQWriter.java

```diff
@@ -196,10 +194,10 @@ private void waitRandomTime() throws InterruptedException {
     Thread.sleep(retryWaitMs + random.nextInt(WAIT_MAX_JITTER));
   }
 
-  private void attemptTableCreate(TableId tableId, String topic) {
+  private void attemptTableCreate(TableId tableId, Set<SinkRecord> records) {
     try {
-      schemaManager.createTable(tableId, topic);
-      logger.info("Table {} does not exist, auto-created table for topic {}", tableId, topic);
+      schemaManager.createTable(tableId, records);
+      logger.info("Table {} does not exist, auto-created table ", tableId);
```

Comment:
Log message should come before `schemaManager.createTable`

Reply:
will update it.